### ISSUE TITLE
Colorpicker editor NullReference error

### ISSUE THREAD

**Issue by flobadea** [umbraco/Umbraco-CMS#5300]
In Umbraco 7.13.2, the colorpicker editor generates an error after clearing the currently selected error and trying to select a new one. 
In the editor controller there is the following check
```
if ($scope.model.value.value === color) {
                $scope.model.activeColor = null;
                $scope.model.value = null;
                angularHelper.getCurrentForm($scope).$setDirty();
                return;
            }
```
after the model.value is set to null, we'll get the error the next time we want to select a color

**Comment by abjerner** [comment]
@flobadea thanks for reporting this 👍 

It does however seem that this has already been fixed in the latest release of Umbraco 7:

https://github.com/umbraco/Umbraco-CMS/commit/62463ecc21c741085100b83bc22caad75dacc7b6

Would it be possible for you to upgrade to Umbraco 7.14.0?

**Comment by nul800sebastiaan** [comment]
I'll close this one as it indeed appears to be fixed in 7.14 already.
